src/main/java/frc/robot/utils/MotorMovement.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.utils;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxLimitSwitch.Type;
+
+public final class MotorMovement {
```

Comment:
I'm not convinced this needs to be its own class. Seems like it would be simple enough to just implement as a private `setMotor()` method on the ArmSubsystem.